.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+    paths-ignore:
+      - '.gitignore'
+      - '**.md'
+      - 'test/'
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+          fetch-depth: 0
+      - name: Extract branch name
+        if: startsWith(github.ref, 'refs/tags/')
+        run: echo "##[set-output name=tag;]${GITHUB_REF#refs/tags/}"
+        id: extract_branch
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build tagged image
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          docker build -t ghcr.io/valhalla/valhalla:${{ steps.extract_branch.outputs.tag }} .
+          docker push ghcr.io/valhalla/valhalla:${{ steps.extract_branch.outputs.tag }}
```

Comment:
i just noticed you removed the `run-` prefixes here and below, now that we dont have a "build" container that we actually publish i think this is fine (our build container is literally just run that deps script)

Reply:
Jep, I also mentioned that in the PR description. I also feel it’s ok to break that now, with a different URL